rosys/hardware/bms.py

```diff
@@ -18,15 +18,18 @@ class Bms(Module, abc.ABC):
     The BMS module provides measured voltages as an event.
     """
 
-    def __init__(self, **kwargs) -> None:
+    def __init__(self, *, battery_low_threshold: float | None = None, **kwargs) -> None:
```

Comment:
Intuitively I would have expected the threshold to be in volts, but according to the implementation it is in percent. This shows how important it is to document all parameters. 🫢